open_dread_rando/game_patches.py

```diff
@@ -51,3 +53,31 @@ def apply_game_patches(editor: PatcherEditor, configuration: dict):
             },
             "mapProps"
         )
+    
+    if configuration["linear_damage_runs"] and configuration["linear_dps"] > 0:
```

Comment:
This means that setting DPS to 0 with the linear option enabled won't edit the environmental damage at all. Seems odd

Reply:
I tried to make it so that a value of zero was not possible to input using the randovania UI. A value of 0 would break the calculations I've made, but I suppose I could just add a special case to disable the damage zones if the value is 0?